Makefile

```
@@ -15,11 +15,13 @@ FONT_AWESOME_TARGET        := themes/$(DOCSY_COMMIT_FOLDER)/assets/vendor/$(FONT
 
 DEV_IMAGE_REGISTRY_NAME    := fluxcd
 HUGO_VERSION               ?= 0.78.2
-DEV_IMAGE_BASE_NAME        := website:hugo-$(HUGO_VERSION)-extended
-PREVIEW_IMAGE_BASE_NAME    := website:hugo-support
-DEV_IMAGE_NAME             := $(DEV_IMAGE_REGISTRY_NAME)/$(DEV_IMAGE_BASE_NAME)
-PREVIEW_IMAGE_NAME         := $(DEV_IMAGE_REGISTRY_NAME)/$(PREVIEW_IMAGE_BASE_NAME)
+HUGO_IMAGE_BASE_NAME       := website:hugo-$(HUGO_VERSION)-extended
+SUPPORT_IMAGE_BASE_NAME    := website:hugo-support
+HUGO_IMAGE_NAME            := $(DEV_IMAGE_REGISTRY_NAME)/$(HUGO_IMAGE_BASE_NAME)
+SUPPORT_IMAGE_NAME         := $(DEV_IMAGE_REGISTRY_NAME)/$(SUPPORT_IMAGE_BASE_NAME)
```

```suggestion
HUGO_IMAGE_NAME            ?= $(DEV_IMAGE_REGISTRY_NAME)/$(HUGO_IMAGE_BASE_NAME)
SUPPORT_IMAGE_NAME         ?= $(DEV_IMAGE_REGISTRY_NAME)/$(SUPPORT_IMAGE_BASE_NAME)
```

maybe, to allow for overwrites?